build

```diff
@@ -4,9 +4,8 @@ set -e
 
 GO="godep go"
 
-ORG_PATH="github.com/kelseyhightower"
-REPO_PATH="${ORG_PATH}/confd"
-
 echo "Building confd..."
+OLD=`pwd`
+cd confd
```

Comment:
This assumes that you're one directory below `confd`. I think it'd be better to use `CONFD_ROOT=$(cd $(dirname $0); pwd)` which will give you the absolute path to this directory.